package.json

```diff
@@ -28,7 +28,7 @@
         "@testing-library/cypress": "^9.0.0",
```

Comment:
When upgrading Cypress version to 13.6.4, this needs to be upgraded as well to 

`"@testing-library/cypress": "^10.0.1",`